modules/python/Makefile.am

```diff
@@ -69,6 +70,28 @@ modules_python_libmod_python_la_DEPENDENCIES = $(MODULE_DEPS_LIBS)
 
 modules/python mod-python: modules/python/libmod-python.la
 
+else
+
+module_LTLIBRARIES += modules/python/libmod-python2.la
```

Comment:
[optional]
I would prefer to keep the `libmod-python.la` name, and define as `libmod-python2.la` (almost arbitrary choose)
Because some cases it is a possibility to have both `libmod-python{,2,3}.la`, where at least two of them defines the same module name as well. (optional because I only could come up the case when developers do switching between branches options)

Reply:
Good idea.